Tests/SonarQube.Bootstrapper.Tests/BootstrapperExeTests.cs

```diff
@@ -200,6 +201,30 @@ public void Exe_PreProc_VersionCheckSucceeds_PreProcSucceeds()
         }
 
         [TestMethod]
+        public void Exe_PostProc_ExecutableNotFound_PostProcFails()
+        {
+            // Arrange
+            string rootDir = TestUtils.CreateTestSpecificFolder(this.TestContext);
+            using (InitializeNonTeamBuildEnvironment(rootDir))
+            {
+                string binDir = CalculateBinDir(rootDir);
+
+                MockBuildAgentUpdater mockUpdater = CreateValidUpdater(binDir, "http://anotherHost");
+                
+                // Act
+                TestLogger logger = CheckExecutionFails(mockUpdater, "end");
+
+                IBootstrapperSettings settings;
+                ArgumentProcessor.TryProcessArgs(new string[] { "end" }, logger, out settings);
```

Comment:
@HSAR why is the test processing the arguments (line 218) after it has already attempted to execute the process with the same arguments in line 215? This test class is testing the bootstrapper, not the ArgumentProcessor.